fets/model/README.md

```diff
@@ -0,0 +1,178 @@
+# FeTS Challenge task 2 - MLCube integration - Model
+
+The FeTS challenge 2022 task 2 focuses on how segmentation methods can learn from multi-institutional datasets to be robust to distribution shifts at test-time, effectively solving a domain generalization problem. In this repository, you can find information on the container submission.
+
+In the FeTS challenge task 2, participants can submit their solution in the form of an MLCube docker image. Note that we do not impose restrictions on the participants how they train their model nor how they perform inference, as long as the resulting algorithm is compatible with the interface described here. After training a model, the following steps are required to submit it:
+
+1. Update the MLCube template with your custom code and dependencies ([guide below](#how-to-modify-this-project)).
+2. Build and test the docker image as described [below](#tasks-execution).
+3. Submit the container as described on the [challenge website](https://www.synapse.org/#!Synapse:syn28546456/wiki/617255).
+
+To make sure that the containers submitted by the participants also run successfully on the remote institutions in the FeTS federation, we will offer functionality tests on a few toy cases. Details are provided in the [challenge website](https://www.synapse.org/#!Synapse:syn28546456/wiki/617255). Note that we will internally convert the submitted docker images into singularity images before running the evaluation.
+
+## Project setup
+
+Please follow these steps to get started:
+<!-- TODO singularity stuff once it is ready -->
+- Install [docker](https://docs.docker.com/engine/install/). You may also have to install the [NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#installing-on-ubuntu-and-debian) for GPU-support.
+<!-- - (Optional) Install [singularity](https://sylabs.io/guides/latest/user-guide/quick_start.html#quick-installation-steps). Only required if you want to test docker-to-singularity conversion yourself. -->
+- [Install MLCube](https://mlcommons.github.io/mlcube/getting-started/) (with docker runner) to a virtual/conda environment of your choice. For example:
+
+```bash
+# Create Python environment and install MLCube Docker runner 
+virtualenv -p python3 ./env && source ./env/bin/activate && pip install mlcube-docker
+```
+
+<!-- - (Optional) Install MLCube's singularity runner. -->
+- Clone this repository
+<!-- TODO Update this once merged -->
+
+```bash
+git clone https://github.com/mlcommons/mlcube_examples && cd ./mlcube_examples
+git fetch origin pull/39/head:feature/fets && git checkout feature/fets
+cd ./fets/model/mlcube
+```
```

Comment:
I would suggest to update this so that it works after merging, too. Not sure if it's best to do it before or after the merge.

```suggestion
- Clone this repository

```bash
git clone https://github.com/mlcommons/mlcube_examples
cd ./mlcube_examples/fets/model/mlcube
```

```

Reply:
Done!
Thanks for pointing this out.